Added events filter to my events page on eventyay-common

yaxit24 opened the pull request:
[#662](https://github.com/fossasia/eventyay-tickets/issues/662) Inserted {% bootstrap_field filter_form.status layout='inline' %} between the query and organizer fields

<img width="1506" alt="Screenshot 2025-06-04 at 8 36 26 AM" src="https://github.com/user-attachments/assets/914ab9ab-d48d-4ec5-b302-f5da73303641" />

## Summary by Sourcery

New Features:
- Add status filter to events page filter form

hpdang commented:
@yaxit24 thanks for the PR. I can't see the screenshot and have difficulty understand what we try to do here. Is there an issue associated with this PR?

hpdang commented:
@yaxit24 I can't see the screenshot. If you put the issue number in the PR description (not in the title) that would make it easier for reviewers

yaxit24 commented:
Thank-you. Yes, @hpdang can i raise a PR for it ?

Gagan-Ram reviewed (approved):
Functionality works as expected.

Saksham-Sirohi reviewed (changes requested):
why are there so many changes i recommend deleting the last 2 commits from your branch and force pushing it again I don't why there are so many unrelated changes introduced

hpdang commented:
This PR fixed the same issue and it works for me, can someone else review this and we can merge?  https://github.com/fossasia/eventyay-tickets/pull/669

yaxit24 commented:
Thank you for the detailed feedback. I’ve removed all unrelated changes and force-pushed the branch with only the necessary commits addressing the event filter addition.The functionality works as expected and I’ve addressed the previous concerns about unrelated files.

Regarding the failing style and spellcheck tests, I have checked them and found that some check-failure are caused by whitespace lines and formatting issues that need fixing. Others may be related to cache versioning and isort configuration.

Please let me know if there’s anything else I should update or improve.

mariobehling reviewed (changes requested):
All events and all organizers use different drop-down buttons. Expected: Follow existing styles and designs of the platform and specific page.
